.github/workflows/lint.yml

```diff
@@ -14,7 +14,12 @@ jobs:
     name: readme
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v4
+      - name: Fix git safe.directory when run inside container
+        # https://github.com/actions/runner/issues/2033
+        run: |
+          mkdir -p /home/runner/work/_temp/_github_home
+          printf "[safe]\n\tdirectory = /github/workspace" > /home/runner/work/_temp/_github_home/.gitconfig
```

Comment:
@Nuru should we just add this directly to the `Dockerfile`?

Reply:
We shouldn't be doing it this way at all.

Fixed.